Update walkthrough (#8)
lightstep/python-opentelemetry-walkthrough: README.rst

In OpenTelemetry, there is a concept of a global tracer for everyone to access.

> Accessing this global tracer is easy, just add these lines to ``server.py``

Reviewer: ```suggestion
Accessing this global tracer can be done by adding these lines to ``server.py``
```
